packages/marshal/src/rankOrder.js

```diff
@@ -46,9 +46,46 @@ const { entries, fromEntries, setPrototypeOf, is } = Object;
  */
 const sameValueZero = (x, y) => x === y || is(x, y);
 
+/**
+ * @param {any} left
+ * @param {any} right
+ * @returns {RankComparison}
+ */
 export const trivialComparator = (left, right) =>
   // eslint-disable-next-line no-nested-ternary, @endo/restrict-comparison-operands
   left < right ? -1 : left === right ? 0 : 1;
```

Comment:
Now that this is no longer meant for strings, we should probably rename it to something like `numericComparator` (possibly with an interim deprecation period) and tighten up the JSDoc types.

Reply:
Again, since it is already exported from the package, I'm electing not make this minor name improvement. The old name is good enough. Resolving.